include/fmt/ranges.h

```diff
@@ -309,6 +309,16 @@ OutputIt write_range_entry(OutputIt out, const Arg& v) {
   return write<Char>(out, v);
 }
 
+template <typename CharT, CharT... C> struct string_literal {
+  static constexpr CharT value[] = {C...};
```

Comment:
Please add array size for EDG based compilers. Details in PR https://github.com/fmtlib/fmt/pull/2982

```c++
static constexpr CharT value[sizeof...(C)] = {C...};
```

Reply:
Done.